pkg/minikube/cni/cni.go

```diff
@@ -150,6 +141,15 @@ func chooseDefault(cc config.ClusterConfig) Manager {
 		return KindNet{cc: cc}
 	}
 
+	if cc.KubernetesConfig.ContainerRuntime != "docker" {
```

Comment:
I am confused, is this just moving the code down ?

Reply:
@medyagh yes, that is correct - before, it would recommend 'bridge' for non-docker cr, even for multinode, and because of https://github.com/kubernetes/minikube/blob/8ce6326186df1b760af03739f38375b7d00d8053/pkg/minikube/cni/bridge.go#L85 it will always fail
this ensures that first it checks if it is multinode and if so - recommend kindnet